src/components/Reports/Reports.test.js

```diff
@@ -73,51 +68,42 @@ describe('Reports page', () => {
     };
 
     // Act
-    const { getByText, container } = render(
-      <AppServicesProvider
-        forcedServices={{
-          datahubClient: datahubClientDouble,
-          appConfiguration: { dopplerLegacyUrl: 'http://test.localhost' },
-        }}
-      >
+    const { queryByText, container } = render(
+      <AppServicesProvider forcedServices={dependencies}>
```

Comment:
<img width="617" alt="image" src="https://user-images.githubusercontent.com/1157864/133345456-9b098658-002e-4cf3-a75f-3f1721f77b47.png">

How does this change work?

Before:

```js
forcedServices = {
  datahubClient: { 
      getAccountDomains: ... // 👈
  },
  appConfiguration: { dopplerLegacyUrl: 'http://test.localhost' },
};
```

Now:

```js
forcedServices = { 
      getAccountDomains: ... // 👈
  };
```

So, the `forcedServices` receives now a totally different structure, is it fine?

Reply:
It seems that these changes are a kind of refactoring, it would be nice to have them in a different PR to easily understand the impact of the change and to be used as a reference in further changes.